tests/userschema.test/t09.expected

```diff
@@ -0,0 +1,19 @@
+[select * from test] failed with rc -3 no such table: test@default
+(t=2)
+(t=2)
+(t=2)
+(t=2)
+(tablename='forecast@dcba', username='abcd', READ='N', WRITE='N', DDL='N')
+(tablename='forecast@dcba', username='dcba', READ='Y', WRITE='Y', DDL='Y')
+(tablename='forecast@dcba', username='default', READ='N', WRITE='N', DDL='N')
+(tablename='forecast@dcba', username='mohit', READ='Y', WRITE='Y', DDL='Y')
+(tablename='test@dcba', username='abcd', READ='N', WRITE='N', DDL='N')
+(tablename='test@dcba', username='dcba', READ='Y', WRITE='Y', DDL='Y')
+(tablename='test@dcba', username='default', READ='N', WRITE='N', DDL='N')
+(tablename='test@dcba', username='mohit', READ='Y', WRITE='Y', DDL='Y')
+[select * from test] failed with rc -3 Invalid set command 'user 12345678901234567'
```

Comment:
Why do we get this error on SELECT?

Reply:
We don't send anything to server until the select.

Reply:
Huh..? On failure, I’d expect the SET command to report an error. But, it should probably be dealt in a separate PR.